gateway/src/http_entrypoints.rs

```diff
@@ -297,6 +297,15 @@ enum PowerState {
     A2,
 }
 
+/// Identifier for an SP's component's firmware slot; e.g., slots 0 and 1 for
+/// the host boom flash.
```

Comment:
Just noticed this typo. Presumably you meant 'host boot rom`

Reply:
I do! Thanks, fixed.